llvm/test/CodeGen/NVPTX/ldu-ldg-willreturn.ll

```diff
@@ -0,0 +1,187 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt < %s -O3 -S | FileCheck %s
```

Comment:
Should not use -O3 in tests, except for test/PhaseOrdering. This should be testable with a single IR pass, and then this would go in test/PassName, not test/CodeGen

Reply:
Moved/updated test to use DCE